wrapanapi/msazure.py

```diff
@@ -436,9 +478,15 @@ def remove_netsec_group(self, group_name, resource_group=None):
         self.logger.info("Network Security Group '%s' is removed", group_name)
         return operation.status()
 
-    def list_load_balancer(self):
+    def list_load_balancers(self):
         self.logger.info("Attempting to List Azure Load Balancers")
-        return [lb.name for lb in self.network_client.load_balancers.list_all()]
+        location = self.region.replace(' ', '').lower()
+        all_lbs = self.network_client.load_balancers.list_all()
+        lbs_in_location = []
+        for lb in all_lbs:
+            if lb.location == location:
+                lbs_in_location.append(lb.name)
+        return lbs_in_location
 
     def does_load_balancer_exist(self, lb_name):
```

Comment:
like mshriver said, looks like changing the method from 'list_load_balancer' to 'list_load_balancers' would break this 'does_load_balancer_exist' method here :)

Reply:
Ok i will change back